### PR TITLE
DAC6-2194: Migrate Second Contact Phone page from Nunjucks to Twirl

### PR DESCRIPTION
**Pull request by ers2015112**

### CODE REVIEW

File: app/controllers/SndContactPhoneController.scala [hunk 1]
```diff
@@ -43,32 +39,23 @@ class SndContactPhoneController @Inject() (
   standardActionSets: StandardActionSets,
   formProvider: SndContactPhoneFormProvider,
   val controllerComponents: MessagesControllerComponents,
-  renderer: Renderer
+  view: SndContactPhoneView
 )(implicit ec: ExecutionContext)
     extends FrontendBaseController
     with I18nSupport
-    with NunjucksSupport {
+    with NunjucksSupport
```
Comment (naree-hmrc) [inline app/controllers/SndContactPhoneController.scala:46]: Do we still need NunjucksSupport?

**Review by naree-hmrc — DISMISSED** [review]
LGTM

File: app/controllers/SndContactPhoneController.scala (hunk 1 above)
Comment (ers2015112) [reply app/controllers/SndContactPhoneController.scala:46]: Probably not I'll remove it

**Review by naree-hmrc — APPROVED** [review]
LGTM

**Pull request merged**